### PR TITLE
Bandage upgrade

### PR DESCRIPTION
**Pull request by neoformit**
Upgrade bandage tool to use the new `bandage_nc` package. This [fork of Bandage](https://github.com/asl/BandageNG) is better maintained with performance upgrades.

~~N.B. failing file size limits, but these files are already in IUC.~~

N.B. Test output (PNG/SVG/JPG) sizes vary a lot - apparently the tools is no longer deterministic.

### CODE REVIEW

File: tools/bandage/bandage_image.xml [hunk 1]
```diff
@@ -69,7 +69,7 @@ Bandage
             <param name="output_format" value="png"/>
             <output name="outfile" ftype="png">
               <assert_contents>
-                  <has_size value="2752" delta="1000"/>
+                  <has_size value="2752" delta="4000"/>
```
Comment (bgruening) [inline tools/bandage/bandage_image.xml:72]: the filesize is only 2700 und the delta is 4000 are we sure those files are generated or correct?

Comment (neoformit) [reply tools/bandage/bandage_image.xml:72]: No, I was tired last night and this probably doesn't make sense :joy:

Comment (bgruening) [reply tools/bandage/bandage_image.xml:72]: I'm getting tired now :) haha

Thanks for working on all of that Cam!

Comment (neoformit) [reply tools/bandage/bandage_image.xml:72]: All good, just wanna make sure this gets through :)

**Review by bernt-matthias — DISMISSED** [review]
Good upgrade, but I would prefer to rename the tool as well.

**Comment by neoformit** [comment]
> Good upgrade, but I would prefer to rename the tool as well.

@bernt-matthias  Like this?

```xml
<tool id="bandage_image" name="BandageNG Image" version="@TOOL_VERSION@+galaxy4" profile="21.05">
```

**Comment by bernt-matthias** [comment]
More like 

```xml
<tool id="bandage_ng_image" name="BandageNG Image" version="@TOOL_VERSION@+galaxy4" profile="21.05">
```

Making a complete new suite.

**Comment by bgruening** [comment]
@bernt-matthias I think it's fine as it is. The old tool is not maintained and the new is a friendly fork, for the user the functionality does not change and keeping it will make workflow updates and tutorials easier to migrate.

**Comment by neoformit** [comment]
Ok so new name, same ID? Or same name too? @bgruening @bernt-matthias

**Comment by bgruening** [comment]
I would keep the ID as it is.

**Comment by neoformit** [comment]
Agreed, but currently the tools have been changed to `name="BandageNG Image"`. So it can be merged like this or I can roll the name back too. I don't think a name change has implications for workflows etc?

**Comment by bgruening** [comment]
The name has no implications. I will wait for @bernt-matthias to merge maybe he has strong feelings about the ID.

**Comment by bernt-matthias** [comment]
You convinced me. We can keep the ID and name if you like.

**Comment by neoformit** [comment]
@bgruening @bernt-matthias what's the action here? Do you want to merge with `name="BandageNG *"` or revert to `name="Bandage *"`?

**Comment by neoformit** [comment]
Also - Gareth has pointed out that the input data for Bandage should probably be `gfa` datatype, not `tabular`. Any thoughts on that? Or why it was done this way originally?

**Comment by bgruening** [comment]
I'm fine with keeping the name as it is now. Gfa looks more accurate isn't it. Or is anything else supported?

**Comment by bernt-matthias** [comment]
> Also - Gareth has pointed out that the input data for Bandage should probably be gfa datatype, not tabular. Any thoughts on that? Or why it was done this way originally?

Galaxy seems to support fastg and gfa (as gfa1 or gfa2). Not sure what the others are. Probably we need to keep `txt` to allow for the others. Still listing `fastg,gfa[12],txt` would increase usability a bit (since the user sees what is allowed in the form field .. in addition to the help text).

Also lets revert name and id of the tool to the original values.

**Review by bgruening — APPROVED** [review]

**Pull request merged**